system/jlib/jfile.hpp

```diff
@@ -632,4 +632,7 @@ const static bool filenamesAreCaseSensitive = false;
 const static bool filenamesAreCaseSensitive = true;
 #endif
 
+extern jlib_decl IDirectoryIterator *getSortedDirectoryIterator(IFile *directory, SortDirectoryMode mode, bool rev, const char *mask = NULL, bool sub = false, bool includedirs = false);
+extern jlib_decl IDirectoryIterator *getSortedDirectoryIterator(const char *dirName, SortDirectoryMode mode, bool rev, const char *mask = NULL, bool sub = false, bool includedirs = false);
```

Comment:
nullptr preferable.

Reply:
Would make sense to default mode and rev here too, since others have defaults and common usage is going to be mode=SD_byname and rev=false